README.md

```diff
@@ -518,12 +605,16 @@ ilab model train
 
 > **NOTE:** ⏳ This step can potentially take **several hours** to complete depending on your computing resources. Please stop `ilab model chat` and `ilab model serve` first to free resources.
 
+If you are using `ilab model train --legacy` or are on MacOS:
+
 `ilab model train` outputs a brand-new model that can be served in the `models` directory called `ggml-model-f16.gguf`.
 
-```shell
- (venv) $ ls models
- ggml-merlinite-7b-lab-Q4_K_M.gguf  ggml-model-f16.gguf
-```
+If you are using `ilab model train` with a GPU enabled system:
+
+`ilab model train` outputs brand-new models that can be served in the `~/.local/share/instructlab/checkpoints` directory.  These models can be run through `ilab model evaluate` to choose the best one.
+
+
+If you are using `ilab model train --strategy lab-multiphase`
```

Comment:
lost some text?

Reply:
the ls models is outdated and not very helpful so I removed it, can add it back if others disagree!

Reply:
I think @booxter is referring to the fact this says 
```text
If you are using `ilab model train --strategy lab-multiphase`
```
followed by nothing, which makes no sense